package.json

```diff
@@ -62,6 +62,7 @@
     "build": "craco build",
     "test": "REACT_APP_CHAIN=0x1 craco test",
     "lint:es": "eslint . --ext .ts,.tsx,.js --max-warnings 0",
+    "lint:es:fix": "eslint --fix . --ext .ts,.tsx,.js --max-warnings 0",
```

Comment:
chore (blocking): remove this, it's no needed. When using yarn arguments are passed down so `yarn lint:es --fix` is the same as this 
```suggestion
```

Reply:
done